### PR TITLE
Smol changes and some improvements.

### PR DESCRIPTION
**Pull request by bluepanee**
Changes:
 - **VeinMiner** module has whitelist mode & delay slider now. #1883 
image

 - **Sneak** can works together with **Freecam**. #1800

 - Ability to disable chat feedback for certain modules. No more spam, Zoom-ers. #1825
image

 - In the **Zoom** module, you can now turn off the dumb hands rendering.
image

### CODE REVIEW

File: src/main/java/meteordevelopment/meteorclient/systems/modules/world/VeinMiner.java [hunk 1]
```diff
@@ -139,7 +151,14 @@ private boolean isMiningBlock(BlockPos pos) {
     @EventHandler
     private void onStartBreakingBlock(StartBreakingBlockEvent event) {
         BlockState state = mc.world.getBlockState(event.blockPos);
-        if (state.getHardness(mc.world, event.blockPos) < 0 || blacklist.get().contains(state.getBlock())) return;
+
+        if (mode.get() == ListMode.Whitelist) {
+            if (!selectedBlocks.get().contains(state.getBlock()))
```
Comment (C10udburst) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/world/VeinMiner.java:156]: the `state.getHardness(mc.world, event.blockPos) < 0` check should be done regardless if the mode is on whitelist or blacklist

**Review by C10udburst — CHANGES_REQUESTED** [review]

File: src/main/java/meteordevelopment/meteorclient/systems/modules/world/VeinMiner.java (hunk 1 above)
Comment (bluepanee) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/world/VeinMiner.java:156]: But why? If I whitelist a block that has a hardness less than zero, doesn't that mean I intend to break it? Of course, if I'm in creative mode or have the ability to do so.

**Comment by RedCarlos26** [comment]
Based pr

File: src/main/java/meteordevelopment/meteorclient/systems/modules/world/VeinMiner.java (hunk 1 above)
Comment (bluepanee) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/world/VeinMiner.java]: Done.

**Comment by RickyTheRacc** [comment]
please add this I beg you

**Comment by RedCarlos26** [comment]
> please add this I beg you

Yes I beg as well

**Comment by arlomcwalter** [comment]
i dont beg

**Comment by arlomcwalter** [comment]
the state of meteor client

**Comment by Bluscream** [comment]
I demand

**Comment by RedCarlos26** [comment]
I mean why would it not be added. It's so useful

**Comment by RickyTheRacc** [comment]
> the state of meteor client

I DEMAND AND REQUIRE that you merge this pr now. Also hello seasnail

**Comment by arlomcwalter** [comment]
> > the state of meteor client
> 
> 
> 
> I DEMAND AND REQUIRE that you merge this pr now. Also hello seasnail

hello

**Comment by Wide-Cat** [comment]
wtf is that the real beabail

**Comment by MineGame159** [comment]
1. Why is the chat feedback field named `toggleChatFeedback` in NBT and not just `chatFeedback`
2. I think by default chat feedback should be enabled for modules
3. The check for module's chat feedback setting should be moved to the `Module.sendToggleMsg()` method instead it being checked manually every time that method is called

Apart from these things it looks fine

**Comment by arlomcwalter** [comment]
> wtf is that the real beabail

possibly

**Comment by bluepanee** [comment]
> 1. Why is the chat feedback field named `toggleChatFeedback` in NBT and not just `chatFeedback`
> 2. I think by default chat feedback should be enabled for all modules
> 3. The check for module's chat feedback setting should be moved to the `Module.sendToggleMsg()` method instead of it being checked manually every time that method is called
> 
> Apart from these things it looks fine

Done.

**Comment by RedCarlos26** [comment]
Less go!!

**Comment by arlomcwalter** [comment]
the state of meteor client

**Pull request merged**